media-video/pipewire/pipewire-0.3.24.ebuild

```diff
@@ -0,0 +1,228 @@
+		$(meson_feature pipewire-alsa) # Allows integrating ALSA apps into PW graph
+		$(meson_feature pipewire-jack) # Allow integrating JACK apps into PW graph
+#		-Dlibjack-path="" # Where to install libjack.so et al (if an absolute path is used, remember to prefix it with ${EROOT} or similar!); setting this will also break pw-jack's multilib support (but presumably that's okay as the intended use would be to replace system's libraries making the loader irrelevant)
+		-Dspa-plugins=enabled
+		-Dalsa=enabled # Allows using kernel ALSA for sound I/O (-Dmedia-session depends on this)
+		-Daudiomixer=enabled # Matches upstream
+		-Daudioconvert=enabled # Matches upstream
+		$(meson_feature bluetooth bluez5)
+		$(meson_feature bluetooth bluez5-backend-hsp-native)
+		$(meson_feature bluetooth bluez5-backend-hfp-native)
+		$(meson_feature bluetooth bluez5-backend-ofono)
+		$(meson_feature bluetooth bluez5-backend-hsphfpd)
+		$(meson_feature aac bluez5-codec-aac)
+		$(meson_feature aptx bluez5-codec-aptx)
+		$(meson_feature ldac bluez5-codec-ldac)
+		-Dcontrol=enabled # Matches upstream
+		-Daudiotestsrc=enabled # Matches upstream
+		$(meson_feature extra ffmpeg) # Disabled by upstream
+		$(meson_feature jack) # Allows PW to act as a JACK client
+		-Dsupport=enabled # Miscellaneous/common plugins, such as null sink
+		-Devl=disabled # Matches upstream
+		$(meson_feature debug test) # fakesink and fakesource plugins
+		$(meson_feature v4l v4l2)
+		-Dlibcamera=disabled # libcamera is not in Portage
+		-Dvideoconvert=enabled # Matches upstream
+		-Dvideotestsrc=enabled # Matches upstream
+		-Dvolume=enabled # Matches upstream
+		$(meson_feature debug vulkan) # Uses pre-compiled Vulkan compute shader to provide a CGI video source (disabled by upstream)
+		$(meson_feature extra pw-cat)
+		-Dudev=enabled
+#		-Dudevrulesdir="" # Directory for udev rules (defaults to /lib/udev/rules.d)
+#		-Dsystemd-user-unit-dir="" # Directory for user systemd units (defaults to pkg-config systemd --variable=systemduserunitpath ; prefix awareness expected but not verified)
+		$(meson_feature debug sdl2) # Enables SDL2 dependent code (currently only examples when -Dinstalled_tests=enabled)
+		$(meson_feature extra sndfile) # Enables libsndfile dependent code (currently only pw-cat)
+		# Gentoo added options
+	)
+	meson_src_configure
+}
+
+src_compile() {
+	meson_src_compile
+
+	einfo "Generating ${limitsdfile}"
+	cat > ${limitsdfile} <<- EOF || die
+		# Start of ${limitsdfile} from ${P}
+
+		1000:60000	-	memlock 256
+
+		# End of ${limitsdfile} from ${P}
+	EOF
+}
+
+src_install() {
+	meson_src_install
+
+	insinto /etc/security/limits.d/
+	doins ${limitsdfile}
+
+	if use pipewire-alsa; then
+		keepdir /etc/alsa/conf.d
+
+		dosym8 -r /usr/share/alsa/alsa.conf.d/50-pipewire.conf /etc/alsa/conf.d/50-pipewire.conf
+		dosym8 -r /usr/share/alsa/alsa.conf.d/99-pipewire-default.conf /etc/alsa/conf.d/99-pipewire-default.conf
```

Comment:
@a17r , unless I have missed eapi8-dosym.eclass being considered bad, this ought to satisfy everyone.

Reply:
there was no problem previously either.

Reply:
Thank you for clearing that up.